extension-style/build.gradle.kts

```diff
@@ -27,6 +27,7 @@ dependencies {
   testImplementation(Dependencies.mockk)
   testImplementation(Dependencies.androidxTestCore)
   testImplementation(Dependencies.robolectric)
+  testImplementation(Dependencies.equalsVerifier)
```

Comment:
it depends on `sdk-base` where we also specify this dependency, guess we do not need it here then

Reply:
it's not like `testApi`

I want to refactor a little bit test dependencies